sections/src/StudentSectionCard.js

```diff
@@ -86,10 +87,10 @@ export default function StudentSectionCard({
             {enrolledInThisSection ? (
               <div>Switch to Section {slotText}</div>
             ) : (
-              <a href="#">
+              <span className="btn-link">
```

Comment:
Nooo, not turning an A into a SPAN!
Can you not do <button type="button"> here?

Reply:
the <a> was not correct - it was used only for the font style. It's enclosed inside a <button> created by the ListGroup.Item (line 82), so this change actually fixes an a11y issue, I Think.

Reply:
At least, eslint was yelling at me about it, so I made the change. The span doesn't have an onclick handler, the handler is associated with the enclosing button and works with keyboard input as well

Reply:
Ohhh nice, thanks for clarifying!

Reply:
I'm planning to make eslint + flow merge-blocking soon, so I took the opportunity to fix the issues it flagged here.